### ISSUE TITLE
CrossEntropyLoss함수를 사용하고 싶습니다

### ISSUE THREAD

**Issue by ghost** [keris2020/hackathon#82]
![cre](https://user-images.githubusercontent.com/73578303/98683732-bd292400-23a8-11eb-815a-ae0ec7c59298.JPG)
![cre_](https://user-images.githubusercontent.com/73578303/98683739-be5a5100-23a8-11eb-9f18-f43a6d609af5.JPG)

제가 BCEWithLogitsLoss()함수를 CrossEntropyLoss()함수로  수정했는데, 이런 오류가 뜹니다.
CrossEntropyLoss함수를 사용하기 위해 추가적으로 해줘야할 작업이 있나요? ex)softmax 함수 작업 등
